about.html

```diff
@@ -0,0 +1,123 @@
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+
+        <nav id="navbar" class="header-body">
+            <i id="mobile-menu-close-icon" class="mobile-menu-close-icon fa-solid fa-x"></i>
+            <a class="logo" href="index.html"><img src="./assets/Logopit_1684831391238.png" alt="logo"></a>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="">Programe</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a href=""><button class="header-btn">CC Campaign</button></a></li>
+            </ul>
+        </nav>
+    </header>
+    <!--Header section ends-->
+
+    <main>
+
+     <!-- Headline Section Starst Here -->
+     <section class="headline">
+        <h6 class="headline__greetings-heading">"Hello! Sharing world"</h6>
+        <h1  class="headline__main-heading">Global Tech Summit 2023</h1>
+        <p class="headline-description">The Global Summit brings together the community of  
+            experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+            This year, we hope to expand our invitation list including organizations and individuals.</p>
+        <h6 class="headline-address">Please contact us per Email for any firther questions about Tech Summit 2023</h6>
+        <h6 class="headline-email">email@domain.com</h6>
+    </section>
+    <!-- Headline Section Ends Here -->
+
+     <!-- Logo Selection Section Starts Here -->
+    <!--  -->
+    <section class="logo-selection">
+        <h3>Global Tech Summit 2023 Logo</h3>
+        <hr>
+        <p>The logo of Tech Summit 2023 was decided through logo competition from 7. July - 12. July</p>
+        <div class="logo-selection__logo">
+            <img src="./assets/Logopit_1684831391238.png" alt="">
+        </div>
+    </section>
+    <hr class="hr-full">
+    
+    <!-- Logo Selection Section Ends Here -->
+
+    <!-- Past Summits Section Starts Here -->
+    
+    <section class="past-summits">
+        <h3>See the past Tech Summits</h3>
+        <hr>
+        <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+        <div class="past-summit-container">
+            <img src="./assets/pexels-matheus-bertelli-3321802.jpg" alt="Avatar" class="past-summit-bg">
+            <div class="past-summit-data">
+                <h4>2020</h4>
+                <p>Tech Summit 2020 Warsaw</p></div>
+          </div>
+          <div class="past-summit-container">
+            <img src="./assets/pexels-matheus-bertelli-3321797.jpg" alt="Avatar" class="past-summit-bg">
+            <div class="past-summit-data">
+                <h4>2020</h4>
+                <p>Tech Summit 2020 Aires</p></div>
```

Comment:
- [ ] Hello @suthish777, As stated by the previous reviewer, it would be okay if you follow the behance design as much as possible. Currently the images in your about page `Desktop view` does not aligned the way it should align. The should be side by side and smaller in view as seen in the original design. 👍 👍 

>N.B: Kindly refer back to the comment made by the previous reviewer.